piccolo/columns/column_types.py

```diff
@@ -993,22 +997,59 @@ class Concert(Table):
     timedelta_delegate = TimedeltaDelegate()
 
     def __init__(
-        self, default: TimestamptzArg = TimestamptzNow(), **kwargs
+        self,
+        default: TimestamptzArg = TimestamptzNow(),
+        at_time_zone: ZoneInfo = ZoneInfo("UTC"),
+        **kwargs,
     ) -> None:
         self._validate_default(
             default, TimestamptzArg.__args__  # type: ignore
         )
 
         if isinstance(default, datetime):
-            default = TimestamptzCustom.from_datetime(default)
+            default = TimestamptzCustom.from_datetime(default, at_time_zone)
 
         if default == datetime.now:
-            default = TimestamptzNow()
+            default = TimestamptzNow(tz=at_time_zone)
 
+        self._at_time_zone = at_time_zone
         self.default = default
-        kwargs.update({"default": default})
+        kwargs.update({"default": default, "at_time_zone": at_time_zone})
         super().__init__(**kwargs)
 
+    ###########################################################################
+
+    def at_time_zone(self, time_zone: t.Union[ZoneInfo, str]) -> Timestamptz:
+        """
+        By default, the database returns the value in UTC. This lets us get
+        the value converted to the specified timezone.
+        """
+        time_zone = (
+            ZoneInfo(time_zone) if isinstance(time_zone, str) else time_zone
+        )
+        instance = self.copy()
+        instance._at_time_zone = time_zone
+        return instance
+
+    ###########################################################################
+
+    def get_select_string(
+        self, engine_type: str, with_alias: bool = True
+    ) -> str:
+        select_string = self._meta.get_full_name(with_alias=False)
+
+        if self._at_time_zone != ZoneInfo("UTC"):
+            # SQLite doesn't support `AT TIME ZONE`, so we have to do it in
+            # Python instead (see ``Select.response_handler``).
+            if self._meta.engine_type in ("postgres", "cockroach"):
```

Comment:
Occurrences of `in` or `==` checks like this one seem to be becoming increasingly common. I would recommend using data structures like `Enum` to mitigate typos and rely more on auto-completion while coding.